java-test-samples/apigw-lambda-list-s3-buckets/README.md

```diff
@@ -228,8 +228,11 @@ TODO
 
 ## Implement application tracing
 You can use AWS X-Ray to track user requests as they travel through your entire application. With X-Ray, you can understand how your application and its underlying services are performing to identify and troubleshoot the root cause of performance issues and errors.
+This [Lambda function](./src/main/java/com/example/App.java) handler has been instrumented using AWS X-Ray. Find your Lambda function in the console. Then navigate to Monitor -> Traces and you should see a graph of your X-Ray trace.
```

Comment:
Is it worth mentioning that when running tests at scale, x-ray cost will be incurred?

Reply:
Probably. But, if we are adding that note, I think we should be adding it to other language samples as well to keep it consistent.